docs/3.1/docs/json-api/index.rst

```diff
@@ -358,118 +293,75 @@ The **JSON API** can return one of the following HTTP status codes:
 - 503 - Service Unavailable, ledger server is not running yet or has been shut down
 - 504 - Gateway Timeout, transaction failed to receive its completion within the predefined timeout
 
+
+HTTP JSON API Errors
+********************
+
 When the Ledger API returns an error code, the JSON API maps it to one of the above codes according to `the official gRPC to HTTP code mapping <https://cloud.google.com/apis/design/errors#generating_errors>`_.
 
-If a client's HTTP GET or POST request reaches an API endpoint, the corresponding response will always contain a JSON object with a ``status`` field, and either an ``errors`` or ``result`` field. It may also contain an optional ``warnings`` and/or an optional ``ledgerApiError`` :
+If a client's HTTP GET or POST request reaches an API endpoint, the corresponding response will always contain a JSON object. Either an expected message (corresponding to endpoint) or an  ``error`` object specified as in the example below:
 
 .. code-block:: none
 
     {
-        "status": <400 | 401 | 403 | 404 | 409 | 500 | 503 | 504>,
-        "errors": <JSON array of strings>, | "result": <JSON object or array>,
-        ["warnings": <JSON object> ],
-        ["ledgerApiError": <JSON object> ]
+    "cause" : "The submitted request has invalid arguments: Cannot unassign contract `ContractId(00a7feb291fe1be6289c4d77f3a432b623083ed3f49bf8535f8571aa8bdf68b647ca10122006207a71ac6ed62dae27429b43e456dfbbc411a49d67b7e0067bef4c914c6e8a)`: source and target domains are the same",
+    "code" : "INVALID_ARGUMENT",
+    "context" : {
+        "category" : "8",
+         "definite_answer" : "false",
+          "participant" : "participant1",
+          "test" : "JsonV2Tests",
+         "tid" : "99e67812af315256ef9a02a9fdb94646"
+      },
+    "correlationId" : null,
+    "definiteAnswer" : null,
+    "errorCategory" : 8,
+    "grpcCodeValue" : 3,
+    "resources" : [ ],
+    "retryInfo" : null,
+    "traceId" : "99e67812af315256ef9a02a9fdb94646"
     }
 
 Where:
 
-- ``status`` -- a JSON number which matches the HTTP response status code returned in the HTTP header,
-- ``errors`` -- a JSON array of strings, each string represents one error,
-- ``result`` -- a JSON object or JSON array, representing one or many results,
-- ``warnings`` -- an optional field with a JSON object, representing one or many warnings.
-- ``ledgerApiError`` -- an optional field with a JSON object, representing detail of an error if it was originated from Ledger API.
+- ``cause`` --  a textual message containing readable error reason,
+- ``code`` -- a LedgerAPI error code,
+- ``context`` -- a Ledger API context of an error,
+- ``traceId`` -- telemetry tracing id
+- ``grpcCodeValue`` and ``errorCategory``  - defined in Canton Error Codes.
```

Comment:
What about `correlationId`, `definiteAnswer`, `resources` and `retryInfo`?

Also it looks like some of this data is duplicate in the `context`. Is that call-specific? Is it realistic to tidy it up?